connect/mirror/src/main/java/org/apache/kafka/connect/mirror/DefaultConfigPropertyFilter.java

```diff
@@ -30,21 +30,29 @@ public class DefaultConfigPropertyFilter implements ConfigPropertyFilter {
     
     public static final String CONFIG_PROPERTIES_EXCLUDE_CONFIG = "config.properties.exclude";
     public static final String CONFIG_PROPERTIES_EXCLUDE_ALIAS_CONFIG = "config.properties.blacklist";
+    public static final String USE_DEFAULTS_FROM = "use.defaults.from";
+    private static final String USE_DEFAULTS_FROM_DOC = "Which cluster's defaults (source or target) to use "
+                                                        + "when syncing topic configurations that have default values.";
+    private static final String USE_DEFAULTS_FROM_DEFAULT = "target";
 
     private static final String CONFIG_PROPERTIES_EXCLUDE_DOC = "List of topic configuration properties and/or regexes "
-                                                                + "that should not be replicated.";
+                                                                + "that should not be replicated."
+                                                                + "This setting also applies to the properties with default values"
+                                                                + "if use.defaults.from is set to 'source'.";
```

Comment:
I think we should remove this part since this property now affects the behavior for all properties, regardless of the value of `use.defaults.from`.
```suggestion
                                                                + "that should not be replicated.";
```